gnat2goto/driver/tree_walk.adb

```diff
@@ -830,6 +842,26 @@ package body Tree_Walk is
       return Args;
    end Do_Call_Parameters;
 
+   ------------------------
+   -- Do_If_Expression --
+   ------------------------
+
+   function Do_If_Expression (N : Node_Id) return Irep is
+      Expr : Node_Id := First (Expressions (N));
```

Comment:
This `First` is a function? If yes, does this get shadowed, or is there a different namespace where this resides compared to `First` in line 852?

Reply:
I think it gets shadowed, but yes I should rename it to something else.